src/app/inventory/item-types.ts

```diff
@@ -300,6 +302,11 @@ export interface D1Stat extends DimStat {
   };
 }
 
+export interface D2Stat extends DimStat {
+  /** Base stat without bonus perks applied. Important for armor. */
+  baseValue: number;
```

Comment:
`D1Stat` also has a `base` property - might as well just pull that up to `DimItem` and not have to worry about changing types everywhere.

Reply:
no problem, i just do not know enough about D1 that i could be sure doing this wouldn't interfere somehow. i will swap it out.